src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -40,6 +40,7 @@
 struct RenderRepeater
 {
     int cnt;
+    uint32_t level;
```

Comment:
What do you think if we put this `uint16_t level` in the `RenderContext`.  I guess the code could be more compact & clean.